changelog.d/7680.bugfix

```diff
@@ -0,0 +1 @@
+Require `parameterized` package version to be at least 0.7.0 for the `parameterized_class` decorator.
```

Comment:
given it only affects tests, it's maybe more a `misc`?

Reply:
(bugfix entries should include an idea of when the bug started, and what the symptoms of the bug are, ideally...)